script/Arcueid/Day 7/07_00_ARC07_3G.txt

```diff
@@ -328,7 +328,7 @@ If doing so makes things easier, then it's a no-brainer."
 [sha:4c5321cd66172498335126cb2101166fbf74dd6e]{
 -- Page 451, Offset 17696.
 -- 俺は一度だけ深呼吸をして、眼鏡を外し、街に向かって歩き出した。
-I take a single deep breath, remove my glasses, and walk toward the city.
+I take a deep breath, with my glasses removed, we begin walking toward the city. //Might be too much liberty here please check this edit
```

Comment:
I'd revert - the flow of this sentence isn't really natural, it cuts from an active voice ("I take off") to more passive "with my glasses removed" with only a comma separating the two. The JP flow is also more serial / active (taking off his glasses, rather than jumping to the state of them being removed).